packages/accounts/src/msca/account-loupe/decorator.ts

```diff
@@ -0,0 +1,48 @@
+import type { Address, ISmartContractAccount } from "@alchemy/aa-core";
+import type { Hash } from "viem";
+import { IAccountLoupeAbi } from "../abis/IAccountLoupe.js";
+import type { FunctionReference, IAccountLoupe } from "./types.js";
+
+export const accountLoupeDecorators = (
+  account: ISmartContractAccount,
+  accountLoupeAddress: Address
```

Comment:
this should be the account itself, so you shouldn't need this. you can just do `account.getAddress()` below